packages/interfaces/src/IAppliance.js

```diff
@@ -38,7 +38,7 @@ class IAppliance {
 	 *
 	 * @return {Boolean} Whether the dependencies and configurations properly exist.
```

Comment:
Should this be `{Promise<Boolean>}`? Honestly [not sure](https://stackoverflow.com/a/46733261).

Reply:
A good question, though for this project we've NOT been denoting promises for other async methods so lets maintain that standard and fix it everywhere at once if we decide to switch to that.